content/en/metrics/custom_metrics/historical_metrics.md

```diff
@@ -0,0 +1,453 @@
+
+        StatsDClient Statsd = new NonBlockingStatsDClientBuilder()
+            .prefix("statsd").
+            .hostname("localhost")
+            .port(8125)
+            .build();
+        Statsd.gaugeWithTimestamp("example_metric.gauge_with_timestamp", new Random().nextInt(20), 1205794800, new String[]{"environment:dev"});
+        Statsd.countWithTimestamp("example_metric.count_with_timestamp", new Random().nextInt(20), 1205794800, new String[]{"environment:dev"});
+    }
+}
+```
+{{< /programming-lang >}}
+
+{{< programming-lang lang="go" >}}
+```go
+package main
+
+import (
+	"log"
+  "time"
+
+	"github.com/DataDog/datadog-go/statsd"
+)
+
+func main() {
+	statsd, err := statsd.New("127.0.0.1:8125")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+  ts := time.Date(2008, time.March, 17, 23, 0, 0, 0, time.UTC)
+	statsd.GaugeWithTimestamp("example_metric.gauge_with_timestamp", 12, []string{"environment:dev"}, 1, ts)
+  statsd.CountWithTimestamp("example_metric.count_with_timestamp", 12, []string{"environment:dev"}, 1, ts)
+}
+```
+{{< /programming-lang >}}
+
+{{< programming-lang lang=".NET" >}}
+```csharp
+using StatsdClient;
+
+public class DogStatsdClient
+{
+    public static void Main()
+    {
+        var dogstatsdConfig = new StatsdConfig
+        {
+            StatsdServerName = "127.0.0.1",
+            StatsdPort = 8125,
+        };
+
+        using (var dogStatsdService = new DogStatsdService())
+        {
+            dogStatsdService.Configure(dogstatsdConfig);
+            var random = new Random(0);
+            var dto = new DateTimeOffset(2008, 03, 17, 23, 00, 00, new TimeSpan(0, 0, 0))
+            dogStatsdService.Gauge("example_metric.gauge_with_timestamp", 10, tags: new[] {"environment:dev"}, dto);
+            dogStatsdService.Counter("example_metric.count_with_timestamp", 10, tags: new[] {"environment:dev"}, dto);
+        }
+    }
+}
+```
+{{< /programming-lang >}}
+
+{{< /programming-lang-wrapper >}}
+
+## Ingestion latency
+
+Ingesting historical metrics includes some ingestion latencies. These latencies can be caused by the delay associated with the metric timestamp.
+
+| Metric Delayed by:   | Ingestion Latency                     |
+|----------------------|---------------------------------------|
+| 1-12 hours           | Near Real-Time Ingestion (1 hour MAX) |
+| 12 hours - 30 days   | Up to 14 hour latency                 |
+| +30 days             | +14 hours latency                     |
+
+{{< img src="metrics/custom_metrics/historical_metrics/diagram_historical-metrics-ingestion_latency.png" alt="Diagram showing how Historical Metrics can take longer to ingest depending on the metric timestamp">}}
+
+## Ingestion billing
+
+Historical Metrics are billed under the Custom Metrics billing SKU. Billable custom metrics are determined by the timestamp of the metrics submitted, regardless of whether they are submitted today or 12 months later. For more information, see the [Custom Metrics billing][3] documentation.
```

Comment:
Hmm even I'm confused here. The mainpoint should be that we haven't added any new line item / SKU to customer's contracts -- it solely may impact their billable volume of indexed custom metrics no? And just because a metric has historical metrics ingestion enabled and it's submitting a metric value with a timestamp 3 months ago, as long as that metric name / tag value combination was actively reporting ANY value (regardless of the timestamp), it'd be considered active in the hour that it was actively reporting any value no?

Reply:
```suggestion
Historical Metrics are billed under the Custom Metrics billing SKU. Billable custom metrics are determined by the timestamp of the metrics submitted, regardless of whether they have a timestamp of today or 15 months into the past. As long as that metric name and tag value combination is actively reporting ANY value (regardless of the timestamp), it would be considered active in the hour that it was submitted. For more information, see the [Custom Metrics billing][3] documentation.
```